### DISCUSSION TITLE
how to custom close button text?

### DISCUSSION THREAD

**Discussion by lumine-id** [jerrywu001/vue3-toastify#21] [Q&A] [answered]
Need to custom close button text, and check event onClose by click or by auto close. thanks

**Comment by jerrywu001** [comment, marked as answer]
Maybe this can help you:

https://vue3-toastify.js-bridge.com/usage/custom-close-button.html

https://vue3-toastify.js-bridge.com/usage/define-callback.html

onClose will trigger by `auto close`  or `click on close`

@lumine-id
